development/mimir-microservices-mode/config/mimir.yaml

```diff
@@ -85,6 +85,8 @@ ruler_storage:
     insecure:          true
 
 alertmanager:
+  # Uncomment to use ring based discovery of alertmanagers
+  # service_discovery_mode: "ring"
```

Comment:
I wasn't sure whether to default to ring or dns, but settled on dns since since presumably that's what we'll always run in cloud.